README.md

```diff
@@ -126,6 +144,27 @@ Write code to do the following:
 Write your code here:
 ```ruby
 # code here
+1.
+INSERT INTO lamps(wishes_remaining) VALUES (3);
+INSERT INTO genies(name) VALUES ('Genie');
+
+2.
+class Genie < ActiveRecord::Base
+  has_many :lamps
+end
+
+class Lamp < ActiveRecord::Base
+  belongs_to :genie
+end
+
+3.
+UPDATE lamps SET wishes_remaining = 1 WHERE wishes_remaining = 3
+
+4.
+INSERT INTO genie(name) VALUES ('Jafar') WHERE wishes_remaining = 3
+
+5.
+UPDATE genie SET lamps = nil WHERE name = "Genie"
```

Comment:
It's good to see that you are feeling comfortable with SQL.  Remember, these queries will update ALL the records that match the conditions.  That will update ALL lamps that have 3 wishes, including Jafar's lamp.

Thanks for sharing what you did know about the question.  If you have access to ActiveRecord models, I recommend you use them to access the database, instead of raw sql.  It makes it much easier and less error prone.